src/ripple/ledger/PaymentSandbox.h

```diff
@@ -188,6 +188,13 @@ class PaymentSandbox final
     apply (PaymentSandbox& to);
     /** @} */
 
+    // Return a map of balance changes on trust lines and amount of XRP destoryed.
+    // The low account is be the first account in the key.
+    // If the two accounts are equal, the map contains the total changes in currency
+    // regargless of issuer. This is useful to get the total change in XRP balances.
```

Comment:
I will spell check all the strings and comments in this file, thanks for find these.